.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -166,14 +166,19 @@ jobs:
       if: ${{ env.SCANNER_BUNDLE_VERSION == 'v1' }}
       run: |
         scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" "definitions/${SCANNER_BUNDLE_VERSION}"
-
+    # Starting with ACS 4.8.1, the --split flag is no longer required for v2 bundle.
+    # TODO: backport ROX-26366 changes to 4.8
     - name: Run updater (multi bundle)
       run: |
-        if [ "$SCANNER_BUNDLE_VERSION" = "v1" ]; then
-          scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" --split bundles
-        else
-          scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" bundles
-        fi
+        case "$SCANNER_BUNDLE_VERSION" in
+          v1|v2)
+            echo "Running updater with --split for version $SCANNER_BUNDLE_VERSION"
+            scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" --split bundles
+            ;;
```

Comment:
If we don't bump the vuln version, or change the update to ignore `--split`, in 4.9.x this will break.

Reply:
Are we backporting the removal of `--split` to 4.8.0?